pkg/controller/certificaterequests/venafi/venafi.go

```diff
@@ -88,7 +90,20 @@ func (v *Venafi) Sign(ctx context.Context, cr *cmapi.CertificateRequest, issuerO
 
 	duration := apiutil.DefaultCertDuration(cr.Spec.Duration)
 
-	certPem, err := client.Sign(cr.Spec.CSRPEM, duration)
+	customFields := []certificate.CustomField{}
```

Comment:
Because this is initialised to be non-nil, there is a *slight* change in behaviour of our usage of the vcert library with this patch.

Previously we'd be setting `CustomField` to nil, whereas we'll now set it to `[]certificate.CustomField{}` even if the annotation isn't specified. I'm not sure if vcert distinguishes the two, but it might be worth us avoiding finding out by keeping it `nil` if the annotation isn't set 😄